Pseudo code is code component (#4189)
quicwg/base-drafts: draft-ietf-quic-recovery.md

We now describe an example implementation of the loss detection mechanisms
described in {{loss-detection}}.

> The pseudocode in this section is licensed as Code Components; see the

Reviewer: The agreement in number between "The pseudocode" and "Components" is jarring.  I'd go either....
```suggestion
The pseudocode in this section is licensed as a Code Component; see the
```
or
```suggestion
The sections which follow contain pseudocode.  The contents of these sections are
licensed as Code Components; see the
```
Given that it's multiple blocks, I think keeping it plural (as it originally was) might be better, but I agree on finding a word besides "snippets."

Author: I'm using "segments", see if that works.

> The pseudocode snippets in this section are licensed as Code Components; see the

Reviewer 2: ```suggestion
The pseudocode in this section is licensed as Code Components; see the
```

> copyright notice above.

Reviewer 2: ```suggestion
copyright notice.
```

I get that we can't cite this directly, but I think we're OK not saying "above".

We now describe an example implementation of the congestion controller described
in {{congestion-control}}.

> The pseudocode snippets in this section are licensed as Code Components; see the

Reviewer 2: ```suggestion
The pseudocode in this section is licensed as Code Components; see the
```

> copyright notice above.

Reviewer 2: ```suggestion
copyright notice.
```
